src/components/callstoaction/Button/style.scss

```diff
@@ -0,0 +1,180 @@
+  color: #fff;
+}
+.Button:hover::before {
+  transform: translateY(-50%) translateX(-50%) scale(1);
+  opacity: 1;
+}
+.Button p {
+  margin: 0;
+}
+.Button__label {
+  display: inline-block;
+  font-family: $ehw-sans-serif-fonts;
+  font-size: $ehw-default-font-size;
+  padding: 0 20px;
+  text-transform: uppercase;
+  letter-spacing: 0.15em;
+}
+.Button__icon {
+  transition: all 240ms ease-out;
+  width: 0;
+  opacity: 0;
+  float: right;
+  margin-left: -5px;
+  border-width: 0;
+  border-style: solid;
+  border-left-width: 1px;
+  overflow: hidden;
+}
+.Button--iconLeft .Button__icon {
+  margin-left: 0;
+  border-left-width: 0;
+  border-right-width: 1px;
+  margin-right: -5px;
+  float: left;
+}
+.Button--hasIcon .Button__icon {
+  width: 44px;
+  opacity: 1;
+}
+.Button--cta,
+.Button--primary {
+  line-height: 44px;
+}
+.Button--thin {
+  line-height: 22px;
+}
+.Button--thin .Button__label {
+  text-transform: none;
+  letter-spacing: 0.05em;
+}
+.Button--cta,
+.Button--secondary {
+  border-color: $ehw-green;
+}
+.Button--cta:before,
+.Button--secondary:before {
+  background-color: #00b44d;
+  box-shadow: 0 0 90px 60px #00b44d;
+}
+.Button--cta .Button__icon,
+.Button--secondary .Button__icon {
+  border-color: $ehw-green-light;
+}
+.Button--primary,
+.Button--tertiary {
+  border-color: $ehw-grey-dark;
+}
+.Button--primary:before,
+.Button--tertiary:before {
+  background-color: #666;
+  box-shadow: 0 0 90px 60px #666;
+}
+.Button--primary .Button__icon,
+.Button--tertiary .Button__icon {
+  border-color: $ehw-grey;
+}
+.Button--cta {
+  color: #fff;
+  background-color: $ehw-green;
+}
+.Button--primary {
+  color: #fff;
+  background-color: $ehw-grey-dark;
+}
+.Button--secondary {
+  color: $ehw-green;
+  background-color: transparent;
+}
+.Button--tertiary {
+  color: $ehw-grey-dark;
+  background-color: transparent;
+}
+.Button--primary.Button--reverse {
+  border-color: #fff;
+}
+.Button--primary.Button--reverse:before {
+  background-color: $ehw-grey-light;
+  box-shadow: 0 0 90px 60px $ehw-grey-light;
+}
+.Button--secondary.Button--reverse,
+.Button--tertiary.Button--reverse {
+  border-color: #fff;
+}
+.Button--secondary.Button--reverse:before,
+.Button--tertiary.Button--reverse:before {
+  background-color: $ehw-grey-light;
+  box-shadow: 0 0 90px 60px $ehw-grey-light;
+}
+.Button--secondary.Button--reverse .Button__icon,
+.Button--tertiary.Button--reverse .Button__icon {
+  border-color: #fff;
+}
+.Button--primary.Button--reverse {
+  color: $ehw-grey-dark;
+  background-color: #fff;
+}
+.Button--secondary.Button--reverse {
+  color: #fff;
+}
+.Button--secondary.Button--reverse:hover {
+  color: $ehw-green;
+}
+.Button--tertiary.Button--reverse {
+  color: #fff;
+}
+.Button--tertiary.Button--reverse:hover {
+  color: $ehw-grey-dark;
+}
+.Button--disabled {
+  color: $ehw-grey-dark;
+  border-color: $ehw-grey-dark;
+  background-color: transparent;
+  border-style: dashed;
+  cursor: default;
+}
+.Button--disabled:before {
```

Comment:
Was converted to css from stylus. No real point spending time manually converting it to sass :P